jest.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  roots: [
+    'src/tests'
+  ],
+  setupFilesAfterEnv: [
+    './src/tests/jest-setup.ts'
+  ],
+  preset: 'ts-jest/presets/js-with-babel',
```

Comment:
This preset allows Babel to compile Javascript for test runs, which most closely mirrors what the production build does.